lib/signaling/v2/peerconnection.js

```diff
@@ -1391,6 +1392,10 @@ function updateEncodingParameters(pcv2) {
 
     if (maxBitrate === null || maxBitrate === 0) {
       removeMaxBitrate(params);
+    } else if (utils.isChromeScreenShareTrack(sender.track)) {
```

Comment:
Curious Question: We could continue to use destructure this function from utils at line 37. Just curious to know why this is done differently here?

Reply:
@syerrapragada - I chose to not destructure, so that it can be mocked easily in unit tests  `utils.isChromeScreenShareTrack`. With destructuring the function had to be mocked before this file is loaded. w/o it it can be mocked w/o reloading the file.